<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmDB</class>
 <widget class="QWidget" name="FrmDB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>数据库信息</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="9" column="0">
       <widget class="QPushButton" name="btn_Test">
        <property name="text">
         <string>测试</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="txt_port"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>数据库端口：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>数据库地址：</string>
        </property>
       </widget>
      </item>
      <item row="7" column="4">
       <widget class="QPushButton" name="btn_sql">
        <property name="text">
         <string>执行</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>数据库名称：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="txt_name"/>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="lbl_result">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 0, 0);</string>
        </property>
        <property name="text">
         <string>未成功！</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>测试SQL：</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="btn_con">
        <property name="text">
         <string>测试连接</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="3">
       <widget class="QLineEdit" name="txt_sql">
        <property name="text">
         <string>select * from test</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>用户密码：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="txt_user"/>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="txt_psw"/>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="txt_addr"/>
      </item>
      <item row="5" column="4">
       <widget class="QPushButton" name="btn_save">
        <property name="text">
         <string>保存配置</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>用户名称：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>驱动名称：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="txt_driver"/>
      </item>
      <item row="9" column="2">
       <widget class="QLabel" name="lbl_Error">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
